dfusion_rust_core/src/models/order.rs

```diff
@@ -91,11 +91,9 @@ impl From<Entity> for Order {
         let batch_information = entity
             .get("auctionId")
             .and(entity.get("slotIndex"))
-            .and_then(|_| {
-                Some(BatchInformation {
-                    slot: U256::from_entity(&entity, "auctionId"),
-                    slot_index: u16::from_entity(&entity, "slotIndex"),
-                })
+            .map(|_| BatchInformation {
+                slot: U256::from_entity(&entity, "auctionId"),
+                slot_index: u16::from_entity(&entity, "slotIndex"),
```

Comment:
I saw this change elsewhere.

Reply:
Yep. I was running clippy locally to make sure I didn't have linting errors and realized that this came up more than once. I investigated and realized that clippy was not being run on test code so I made PR #308.